bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java

```diff
@@ -661,7 +636,41 @@ public void transformInsertBeneficiaryEvent() {
 		Assert.assertEquals(record.stateCode, bene.getAddress().get(0).getState());
 		Assert.assertEquals(record.countyCode, bene.getAddress().get(0).getDistrict());
 		Assert.assertEquals(record.postalCode, bene.getAddress().get(0).getPostalCode());
-		// TODO test the rest of the columns/resource once they're all ready
+		Assert.assertEquals(Date.valueOf(record.birthDate), bene.getBirthDate());
+		Assert.assertEquals("MALE", bene.getGender().toString().trim());
+		// TODO Following fields were initially built using extension. Decided
+		// further research needs to
+		// be done so these field mappings are documented in a JIRA ticket
+		// "Finalize fields for Beneficiary"
+		/*
+		 * System.out.println("race is " +
+		 * bene.getExtension().get(0).getValue().toString());
+		 * System.out.println("race url is " +
+		 * bene.getExtensionsByUrl(CODING_SYSTEM_CCW_BENE_RACE_CD).toString());
+		 * Assert.assertEquals(record.race,
+		 * bene.getExtensionsByUrl(CODING_SYSTEM_CCW_BENE_RACE_CD).get(0).
+		 * getValue()); Assert.assertEquals(record.entitlementCodeOriginal,
+		 * bene.getExtensionsByUrl(CODING_SYSTEM_CCW_BENE_ENTLMT_RSN_ORIG).get(0
+		 * ).getValue()); Assert.assertEquals(record.entitlementCodeCurrent,
+		 * bene.getExtensionsByUrl(CODING_SYSTEM_CCW_BENE_ENTLMT_RSN_CURR).get(0
+		 * ).getValue());
+		 * 
+		 * Assert.assertEquals(record.medicareEnrollmentStatusCode,
+		 * bene.getExtensionsByUrl(CODING_SYSTEM_CCW_BENE_ESRD_IND).get(0).
+		 * getValue());
+		 */
+		for (int f = 0; f < bene.getIdentifier().size(); f++) {
+			if (bene.getIdentifier().get(f).getSystem().equals(CODING_SYSTEM_CCW_BENE_CRNT_HIC_NUM)) {
+				Assert.assertEquals(record.hicn, bene.getIdentifier().get(f).getValue());
```

Comment:
This will need to be removed, anyways, per my other comment about the HICN, but if that weren't the case, you should instead be using this class' `assertIdentifierExists` method.